nav2_behavior_tree/plugins/condition/is_path_valid_condition.cpp

```diff
@@ -0,0 +1,52 @@
+
+#include <string>
+#include "nav2_behavior_tree/plugins/condition/is_path_valid_condition.hpp"
+
```

Comment:
The spacing in these file are weird, but that's not really the point in this PR so I'll not mention them all

Reply:
I will fix this

Reply:
Done